py3status/modules/player_control.py

```diff
@@ -10,6 +10,7 @@
 Configuration parameters:
     cache_timeout: how often to update in seconds (default 10)
     debug: enable verbose logging (bool) (default False)
+    format: string to print (default "{icon}")
```

Comment:
D'oh. Copy and paste. Will fix it. Clarify please. Leave it? Rename it to `format of output`? I can't come up with something. `a string to display`, `display string`?